experimental/origin-graphql/src/typeDefs/Growth.js

```diff
@@ -120,7 +120,6 @@ module.exports =
     code: String!
     success: Boolean!
     message: String!
-    invites: [Invite]
```

Comment:
@sparrowDom Please confirm it's ok to delete this - I could not see why it would get returned ?

Reply:
Yes good point remove away

Reply:
Sounds good. Thanks for confirming.